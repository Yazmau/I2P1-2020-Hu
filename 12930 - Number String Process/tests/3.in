499
6886198384768386488942872825835247864599435553414796852636922133595685544112414333193956316449741463751527155153944395716967683939626123193681253759724712988616438563935227983661582116218559381636551713613421477476398363212155935663618553287526978562388658231468652375719165664737663586918373626143711239747462193737757945167589513725789721654765894961712953259169568425863346292718683192539423468262748342627456999999999488328972945766753182376698328813546323318375512171914675583589256114187142772
92
493 496 3
485 489 4
481 483 2
475 478 3
473 474 1
469 471 2
465 467 2
461 463 2
453 457 4
449 451 2
445 447 2
437 441 4
433 435 2
425 429 4
421 423 2
413 417 4
405 409 4
397 401 4
391 394 3
385 388 3
377 381 4
369 373 4
365 367 2
363 364 1
355 359 4
353 354 1
345 349 4
337 341 4
333 335 2
331 332 1
325 328 3
321 323 2
319 320 1
317 318 1
309 313 4
301 305 4
295 298 3
287 291 4
281 284 3
273 277 4
271 272 1
265 268 3
257 261 4
253 255 2
251 252 1
245 248 3
239 242 3
237 238 1
229 233 4
225 227 2
221 223 2
219 220 1
213 216 3
209 211 2
207 208 1
199 203 4
193 196 3
187 190 3
179 183 4
171 175 4
163 167 4
159 161 2
155 157 2
153 154 1
147 150 3
141 144 3
135 138 3
129 132 3
123 126 3
119 121 2
113 116 3
107 110 3
103 105 2
99 101 2
91 95 4
85 88 3
77 81 4
73 75 2
65 69 4
61 63 2
53 57 4
51 52 1
45 48 3
37 41 4
33 35 2
31 32 1
29 30 1
27 28 1
21 24 3
17 19 2
11 14 3
3 7 4
