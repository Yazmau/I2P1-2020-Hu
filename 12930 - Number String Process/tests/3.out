109624
23493257
5336
393525
4
323
660
666
11464218
2268
5727
12635568
5092
8531665
3984
99980001
12002763
12698918
197964
103488
80500093
20297706
2856
45
56391312
2
56682598
39514816
5073
14
302157
3417
36
35
16177761
11376162
101673
6660962
551815
41777664
6
269625
27219192
5330
64
489055
235375
6
12309780
2065
180
6
534204
3619
4
9576512
350436
212979
51055998
52807545
19860558
2432
5368
18
457812
46875
117392
588744
301824
1536
423951
296989
3692
1887
18644562
302096
4575569
264
53044992
455
36430486
48
163339
21308670
4992
8
15
40
80934
8272
590112
23843706
